exercises/sum-of-multiples/description.md

```diff
@@ -1,6 +1,9 @@
 # Description
 
-Given a number, find the sum of all the unique multiples of particular numbers up to but not including that number.
+Given a limit and a list of factors, find the sum of all unique multiples of the factors that are less than the limit.
+All inputs will be greater than or equal to zero.
+
+## Example
```

Comment:
I think maybe it would help if we expanded the example a bit as well.

E.g.

Suppose the limit is 20 and the list of factors is [3, 5]. We need to find the sum of all unique multiples of 3 and 5 that are less than 20.

Multiples of 3 less than 20: 3, 6, 9, 12, 15, 18
Multiples of 5 less than 20: 5, 10, 15

The unique multiples are: 3, 5, 6, 9, 10, 12, 15, 18

The sum of the unique multiples is: 3 + 5 + 6 + 9 + 10 + 12 + 15 + 18 = 78

So, the answer is 78.

Reply:
I agree that that's better :)

Reply:
I agree that that's better :)

Reply:
I like that suggestion!

Reply:
Cool. I have made it so!